ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-release-4.20__periodics.yaml

```diff
@@ -89,6 +89,27 @@ tests:
     - ref: baremetalds-e2e-ovn-bgp-pre
     workflow: baremetalds-e2e-ovn-dualstack
   timeout: 5h0m0s
+- as: e2e-aws-kubevirt-ovn-techpreview
+  cluster: build05
+  cron: 0 0 * * *
+  steps:
+    cluster_profile: aws
+    env:
+      CNV_PRERELEASE_LATEST_CHANNEL: "true"
```

Comment:
Are there KubeVirt releases for 4.20 available right now or not?
I am wondering if we could avoid forgetting to remove this _again_

Reply:
no, there aren't - until CNV 4.20 is released, this is how you consume the latest release.